src/client.cpp

```diff
@@ -65,21 +65,25 @@ void waybar::Client::handleOutput(std::unique_ptr<struct waybar_output> &output)
 
 bool waybar::Client::isValidOutput(const Json::Value &                    config,
                                    std::unique_ptr<struct waybar_output> &output) {
-  bool found = true;
+  auto check_output = [](std::string_view                       config_output_name,
+                         std::unique_ptr<struct waybar_output> &output) {
+    if (config_output_name.length() > 0) {
```

Comment:
```suggestion
    if (!config_output_name.empty()) {
```

Reply:
Can you avoid using another function and don't return when the result is false as it'll skip others screen.

Reply:
WDYM by another function? I've used lambda to reduce duplication of that code.